obsreport/obsreport.go

```diff
@@ -38,23 +38,22 @@ var (
 	okStatus = trace.Status{Code: trace.StatusCodeOK}
 )
 
-// SetParentLink tries to retrieve a span from parentCtx and if one exists
-// sets its SpanID, TraceID as a link to the Span from the provided context.
+// setParentLink tries to retrieve a span from parentCtx and if one exists
+// sets its SpanID, TraceID as a link to the given Span.
 // It returns true only if it retrieved a parent span from the context.
 //
 // This is typically used when the parentCtx may already have a trace and is
 // long lived (eg.: an gRPC stream, or TCP connection) and one desires distinct
 // traces for individual operations under the long lived trace associated to
 // the parentCtx. This function is a helper that encapsulates the work of
 // linking the short lived trace/span to the longer one.
-func SetParentLink(ctx, parentCtx context.Context) bool {
+func setParentLink(parentCtx context.Context, span *trace.Span) bool {
```

Comment:
`span` is the child span, right? Would it be more explicit if we name it `childSpan`?